src/Pages/SignUp/SignUp.js

```diff
@@ -1,9 +1,82 @@
-import React from 'react'
+import React, {useState} from 'react'
+import { Link } from 'react-router-dom'
+import { useAuthContext } from '../../Context'
+import { LoaderOverlay } from '../../Components';
 
 function SignUp() {
+
+  const { signUp, isLoading } = useAuthContext()
+  const [email, setemail] = useState('');
+  const [password, setpassword] = useState('')
+  const [confirmpassword, setconfirmpassword] = useState('')
+  const [errorValues, seterrorValues] = useState({})
+
+  const validateSubmit = () => {
+    const errors = {}
+    const regex = /^[^\s@]+@[^\s@]+\.[^\s@]{2,}$/i;
+
+    if(email.length === 0){
+      errors.email = "Email Required"
+    }else if(!regex.test(email)) {
+      errors.email = "This is not a valid email format!";
+    }
+
+    if(password.length === 0){
+      errors.password = "Password Required"
+    }else if(password.length < 4){
+      errors.password = "Password should be atleast 4 characters long"
+    }
+
```

Comment:
I think you are using firebase. Firebase at least requires 6 char length as a password. So check for at least six characters, otherwise, the user will not be created.

Reply:
yes thanks, i will update that